lego/apps/companies/action_handlers.py

```diff
@@ -53,7 +53,8 @@ def handle_create(self, instance, **kwargs):
             # reply is sent to them (and bedkom)
             recipients.append(mail_context["mail"])
             if mail_context["readme"]:
-                recipients.append(f"lederreadme@{settings.GSUITE_DOMAIN}")
+                # recipients.append(f"lederreadme@{settings.GSUITE_DOMAIN}")
+                recipients.append("redaktor@abakus.no")
```

Comment:
```suggestion
                recipients.append("redaktor@{settings.GSUITE_DOMAIN}")
```
not that it matters much